packages/apps/tools/src/pages/home/index.tsx

```diff
@@ -229,13 +230,13 @@ const Home: FC = () => {
               {t('Frequently Asked Questions')}
             </Heading>
             <Box marginBlockEnd="md" />
-            <Accordion.Root>
-              {faqs.map((faq) => (
-                <Accordion.Section title={faq.title} key={faq.title}>
+            <Accordion items={faqs}>
+              {(faq) => (
+                <AccordionItem title={faq.title} key={faq.title}>
```

Comment:
Before (on current tools.kadena.io ) you could expand both items. Is this no longer possible?

Reply:
you can do it, it was not clear that this is the case, the new Accordion has `selectionMode` by default it is `single` but possible values are `multiple` and `none`, I will change this to `multiple` if that is the required behavior